src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationTest.kt

```diff
@@ -2673,6 +2673,14 @@ class ApplicationTest : IntegrationTestBase() {
               )
             }
 
+            val region = probationRegionEntityFactory.produceAndPersist {
+              withId(submittingUser.probationRegion.id)
+            }
+
+            val pdu = probationDeliveryUnitFactory.produceAndPersist {
+              withProbationRegion(region)
+            }
+
```

Comment:
Presumably this is the right way to insert a PDU and region for integration testing?

Reply:
Yep, i think that'll work fine 👍